Dockerfile.win

```diff
@@ -62,3 +63,5 @@ COPY . /updater2
 RUN set -e; for D in . quazip fluid; do cd /updater2/$D && git clean -dXff; done
 WORKDIR /build
 RUN /qt/bin/qmake -config release QMAKE_LFLAGS+="-static" INCLUDEPATH+=/qt/include/QtZlib /updater2 && make -j`nproc`
+WORKDIR /release-win
+RUN cp /build/release/updater2.exe UnvanquishedUpdater.exe && zip UnvUpdaterWin.zip UnvanquishedUpdater.exe
```

Comment:
Since we already do `Unvanquished Updater.app` with one space for macos, maybe we can do it other OS?

Reply:
We could have the space on Windows too at least, doesn't matter to me one way or the other. You'd have to change the name in the v0.0.5 release and the download link though.

Reply:
Keeping it without the space because `Unvanquished Updater.exe` (with the space and the .extension) looks stupid